frontend/src/component/executiveDashboard/LineChart/LineChartComponent.tsx

```diff
@@ -223,28 +228,38 @@ const customHighlightPlugin = {
 };
 
 const LineChartComponent: VFC<{
-    data: ChartData<'line', (number | ScatterDataPoint | null)[], unknown>;
+    data: ChartData<any, unknown>;
     aspectRatio?: number;
     cover?: ReactNode;
     isLocalTooltip?: boolean;
+    overrideOptions?: ComponentProps<typeof Line>['options'];
```

Comment:
isn't there a named/explicit type in chartjs for that so that we don't have to advanced type level operation?